internal/httpserver/matchers_test.go

```diff
@@ -0,0 +1,31 @@
+package httpserver
+
+import (
+	"regexp"
+
+	gomock "github.com/golang/mock/gomock"
+)
+
+var _ gomock.Matcher = (*regexMatcher)(nil)
+
+type regexMatcher struct {
```

Comment:
is this used for comparing logs in tests? or is it used anywhere else?

Reply:
Oh yeah sorry I forgot to mention, I started using golang/gomock for mocks locally in these 2 packages, I hope you don't mind!

And that's a custom matcher used only for this package's tests, since the port number picked (at random availability) is unpredictable.